Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM messense/rust-musl-cross:x86_64-musl
```

Comment:
nit: Could you add a "as builder" here (or any other name that helps identifying the container), and then refer to it in the COPY, instead of using `0`?